packages/flame_tiled/test/tiled_test.dart

```diff
@@ -406,29 +406,31 @@ void main() {
 
       final world = World(children: [component]);
       final cameraComponent = CameraComponent(world: world);
-      cameraComponent.viewfinder.anchor = Anchor.topLeft;
 
       // Need to initialize a game and call `onLoad` and `onGameResize` to
       // get the camera and canvas sizes all initialized
       final game = FlameGame(children: [world, cameraComponent]);
-      await game.ready();
+      cameraComponent.viewfinder.anchor = Anchor.center;
```

Comment:
@spydon @kurtome @ufrshubham 

So, following up the logic that we need to apply the camera's anchor to the parallax, I thought that, hey, maybe the code previously was already correct, but with the previous camera, tiled was always considering a center camera, hence why it was doing a `/ 2`, which would make the generated snapshots be based on a `Anchor.center` camera.

So for the test to pass, I just changed the camera's anchor on the test to center, to match the previous behaviour, which seems to make sense to me.

Let me know if this also make sense to you all.

Reply:
Seems to make sense!